drivers/can/can_mcp2515.c

```diff
@@ -412,15 +503,23 @@ static void mcp2515_rx_filter(struct device *dev, struct zcan_frame *msg)
 
 static void mcp2515_rx(struct device *dev, u8_t rx_idx)
 {
+	__ASSERT(rx_idx < MCP2515_RX_CNT, "rx_idx < MCP2515_RX_CNT");
+
 	struct zcan_frame msg;
 	u8_t rx_frame[MCP2515_FRAME_LEN];
-	u8_t addr_rx_ctrl = MCP2515_ADDR_RXB0CTRL +
-			    (rx_idx * MCP2515_ADDR_OFFSET_FRAME2FRAME);
+	u8_t nm;
+
+	/* Address Pointer selection */
+	if (rx_idx == 0) {
```

Comment:
could do `nm = 2 * rx_idx`, no comparison and matches the tx approach

Reply:
On a general note I am fine with having a magic number here, because this is a single instance for calculating `nm` and a define would be a long name that needs dereferencing to cross check with the datasheet.

Reply:
Done